src/Infrastructure/Persistence/Doctrine/Migrations/Version20230314103748.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230314103748 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE location DROP CONSTRAINT fk_5e9e89cb9f073263');
+        $this->addSql('DROP INDEX uniq_5e9e89cb9f073263');
+        $this->addSql('ALTER TABLE location ADD regulation_order_uuid UUID DEFAULT NULL');
+        $this->addSql('ALTER TABLE location DROP regulation_condition_uuid');
```

Comment:
Sur le modèle de #222 on pourra(it) migrer les données ici, et ensuite mettre la colonne en non-null (une location est forcément liée à un regultion_order, 1-1)

Je crois que ceci fonctionnerait :

```suggestion
        $this->addSql('
            UPDATE location
            SET regulation_order_uuid = ro.uuid
            FROM regulation_order AS ro
            JOIN regulation_condition AS rc ON ro.uuid = rc.regulation_order_uuid
            WHERE rc.uuid = location.regulation_condition_uuid
        ');
        $this->addSql('ALTER TABLE location ALTER COLUMN regulation_order_uuid SET NOT NULL');
        $this->addSql('ALTER TABLE location DROP regulation_condition_uuid');
```